### ISSUE TITLE
Bug; arrow key in Comment pop-up.

### ISSUE THREAD

**Issue by Laurvin** [revilheart/ESGST#361]
When you open a comment pop-up on a thread and press arrow right in it, it will load the next page of the thread below it and the input box loses focus.

**Comment by revilheart** [comment]
Are you sure it's not just Shortcut Keys?

**Comment by Laurvin** [comment]
Yes, it's cause "Focus on the reply box:" is set to 'arrowright'. I don't remember changing this; it's not a feature I'd use.

So if this is the default it's rather a problem. If you use the reply box instead of the pop-up then pressing arrow right doesn't move the cursor at all. I'll just change the hotkey but it might be worth looking at disabling some hotkeys when an textarea has focus.

**Comment by revilheart** [comment]
Yes, there is, just click the input box and type the key. The default should be Ctrl + Q though, I have no idea how you got to the right arrow.

**Comment by Laurvin** [comment]
It turns out that it does take the change to the input box but doesn't display it. When I select "Focus on the reply box:" and press any key, nothing is shown to happen. But when I close Settings and then open it again it does show the new shortcut key.

I did have some other weird stuff happening but for now I can't be sure what was going on. I'll run some more tests tomorrow.

**Comment by revilheart** [comment]
Is there anything in the console when you press the key?

**Comment by Laurvin** [comment]
Nothing in console.

I tried it again; updated to latest version, opened my user page and then clicked the ESGST button to get settings. Clicked "Close all currently opened popups:"; the text in there ("escape") turns grey. I press '6' on my keyboard but nothing happens.

Then I close settings by eiher clicking next to the popup or clicking the Close link at the bottom. I click the ESGST button again and now it shows "6" at "Close all currently opened popups:". At no time did anything show up in the console. It seems likely this is a Firefox issue but it would be very worthwhile to know if others have this same problem.

I just tried the same on another pc where I have a Firefox with no addons except for Greasemonkey and uBlock Origin (even tried with the page whitelisted) and the exact same thing happens.
